format/pyrdp/pyrdp.go

```diff
@@ -0,0 +1,144 @@
+// Processes PyRDP replay files
+// https://github.com/GoSecure/pyrdp
+//
+// Copyright (c) 2022-2023 GoSecure Inc.
+// Copyright (c) 2024 Flare Systems
```

Comment:
Ok to make this explicitly say under MIT license etc? debian packages fq and they are usually picky about unclear licenses

Reply:
Done